_posts/theme/2021-11-27-D42ker.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: theme
+title: D42ker GitHub
+category: theme
+homepage: https://github.com/LeonWong0609/Typora-D42ker-Github-theme
+download: https://github.com/LeonWong0609/Typora-D42ker-Github-theme
+built-in: false
+author: Leon Wong
+thumbnail: d42ker-typora.png
```

Comment:
this file seems not exist

Reply:
Sorry, wrong file name, just [fixed this](https://github.com/typora/typora-theme-gallery/pull/244/commits/2add366926242de4a46c8b6ae5b0a70b7779b959).